apriltag_center:
- 174
- 434
area:
  max_area: 7000
  min_area: 500
calibration_tag_id: 100
calibration_tag_id_2: 1
calibration_tag_size: 0.025
camera_resolution:
  height: 480
  width: 640
center_pose_cam:
  euler:
  - 164.87451920026845
  - 2.1270198571726935
  - 2.8730682108666032
  xyz:
  - 0.029316243817644385
  - -0.03239278793996899
  - 0.29034544361543924
center_pose_world:
  euler:
  - 0.0
  - 0.0
  - 0.0
  xyz:
  - 0.1863
  - 0.0
  - 0.0
extristric:
- - 0.02186137425872101
  - 0.006504178020274199
  - 0.19317346931600585
- - -0.05365435106811478
  - 0.9965084236458471
  - 0.06397008843458543
- - 0.9859813016207213
  - 0.06300574112564794
  - -0.15450291078007708
- - -0.15799393490184382
  - 0.054783557643485276
  - -0.9859192047760074
hand2cam_tf_matrix:
- - 0
  - 0
  - 1
  - -0.101
- - -1
  - 0
  - 0
  - 0
- - 0
  - -1
  - 0
  - 0.045
- - 0
  - 0
  - 0
  - 1
hand2cam_transformation_matrix:
- - -0.01782217674806108
  - -0.1046008609006226
  - 0.9943545795714974
  - -0.09630850772104702
- - -0.9998308045282818
  - 0.006393502312188604
  - -0.01724776636300579
  - -0.009750665301640829
- - -0.004553277093442498
  - -0.9944937320199825
  - -0.10469710903686735
  - 0.07899833508544073
- - 0.0
  - 0.0
  - 0.0
  - 1.0
image_proc_size:
  height: 240
  width: 320
lab:
  black:
    max:
    - 89
    - 255
    - 255
    min:
    - 0
    - 0
    - 0
  blue:
    max:
    - 172
    - 178
    - 102
    min:
    - 11
    - 119
    - 59
  green:
    max:
    - 202
    - 119
    - 209
    min:
    - 46
    - 68
    - 128
  red:
    max:
    - 185
    - 183
    - 189
    min:
    - 9
    - 142
    - 141
  white:
    max:
    - 255
    - 171
    - 182
    min:
    - 110
    - 101
    - 101
perspective_transformation_matrix:
- 1.3356152544683584
- 0.2826949033415031
- -120.9596869440837
- 0.004259859601720424
- 1.6961561541048225
- -93.53444651853098
- -0.0001429229417776327
- 0.0008898632617431417
- 1.0
perspective_transformation_matrix_inv:
- 0.785877085769113
- -0.1723925189856272
- 78.93480742390472
- 0.004022757399421211
- 0.5822466225825433
- 54.94670705622819
- 0.00010874016095310427
- -0.0005427587246641123
- 1.0
roi:
  height:
  - 0.04
  - 0.87
  width:
  - 0.09
  - 0.95
size:
  height: 240
  width: 320
tag_size: 0.025
waste_target_position:
  food_waste:
  - 0.018
  - -0.235
  - 0.015
  hazardous_waste:
  - -0.045
  - -0.235
  - 0.015
  recyclable_waste:
  - -0.11
  - -0.235
  - 0.015
  residual_waste:
  - 0.085
  - -0.236
  - 0.02
white_area_pixel_size:
  width: 520
white_area_pose_cam:
- - -0.05446305507748935
  - -0.9967387417788919
  - -0.05938401875512419
  - 0.021840154737743752
- - 0.9828376517095324
  - -0.06400654372793246
  - 0.17296921031009152
  - 0.006482959742638538
- - -0.17620801807478936
  - -0.048944145233383564
  - 0.9831184709124381
  - 0.19301761135896534
- - 0.0
  - 0.0
  - 0.0
  - 1.0
white_area_pose_world:
- - -0.997107256901652
  - 0.051138982468478165
  - 0.056096784250176994
  - 0.190129134005811
- - 0.05446305507748952
  - 0.9967387417788919
  - 0.059384018755124036
  - -0.021840154737743776
- - -0.0528794240306781
  - 0.06226693717143116
  - -0.9966410719819327
  - 0.01558708787137425
- - 0.0
  - 0.0
  - 0.0
  - 1.0
white_area_world_size:
  height: 0.135
  width: 0.175
